test/drawing_test.dart

```diff
@@ -70,10 +70,10 @@ void main() {
       final GYWIcon icon = GYWIcons.left.icon;
 
       final GYWDrawing drawing = IconDrawing(
-        icon,
+        icon: icon,
         left: 120,
         top: 220,
-        color: Colors.green,
+        colorHex: Colors.green.value,
```

Comment:
Even if we store the value itself, can't we have the constructor that takes a Color object, it will be easier to use I think ?

Reply:
I have scouted the interwebs and I don't think that's possible. Freezed models are supposed to be PODs (plain old data structures) without OOP features like encapsulation. Otherwise, copyWith would also need to change to avoid leaking private API.